headers/addons/bootsel_button.h

```diff
@@ -23,4 +23,5 @@ class BootselButtonAddon : public GPAddon {
 	uint32_t bootselButtonMap;
 };
 
-#endif  // _BootselButton_H_
\ No newline at end of file
+#endif  // _BootselButton_H_
+
```

Comment:
Try to clean these changes as well, bug fixes and cleanliness like this should be in other PRs